### ISSUE TITLE
Myriad of warnings generated when running make configure

### ISSUE THREAD

**Issue by helenmasters** [AdoptOpenJDK/openjdk-tests#6]
I have been following the instructions in the README and the `make -f run_configure.mk` step generates a large number of warnings. Are these warnings things which the user needs to take account of or can they be removed or suppressed as I feel that this renders the output unusable? (see example output below)

```
helens-mbp:TestConfig helenmasters$ make -f run_configure.mk
cd /Users/helenmasters/openjdk-tests/TestConfig/scripts/testKitGen; \
	perl testKitGen.pl ; \
	cd /Users/helenmasters/openjdk-tests/TestConfig;
projectRootDir is not provided. Set projectRootDir = /Users/helenmasters/openjdk-tests/TestConfig/scripts/testKitGen/../../..
graphSpecs is not provided. Set graphSpecs = aix_ppc,aix_ppc-64,aix_ppc-64_purec,aix_ppc_purec,linux_390,linux_390-64,linux_390-64_cs,linux_390-64_purec,linux_390_purec,linux_arm,linux_ppc-64_le,linux_ppc-64_le_purec,linux_x86,linux_x86-64,linux_x86-64_purec,linux_x86_purec,win_x86,win_x86-64,win_x86-64_purec,win_x86_purec,zos_390,zos_390-64,zos_390-64_purec,zos_390_purec
JCL_VERSION is latest
Warning: Cannot get data from modes service! Getting data from modes.xml and ottawa.csv...
Cannot open modes.xml. You can provide the file with option --modeXml=<path>.
Cannot open ottawa.csv. You can provide the file with option --ottawaCsv=<path>.

Generating make file based on /Users/helenmasters/openjdk-tests/TestConfig/scripts/testKitGen/../../../OpenJDK_Playlist/playlist.xml...
Generated /Users/helenmasters/openjdk-tests/TestConfig/scripts/testKitGen/../../../OpenJDK_Playlist/autoGenTest.mk!

Generating make file based on /Users/helenmasters/openjdk-tests/TestConfig/scripts/testKitGen/../../../TestExample/playlist.xml...
Generated /Users/helenmasters/openjdk-tests/TestConfig/scripts/testKitGen/../../../TestExample/autoGenTest.mk!

Generating make file based on subdirs...
Generated /Users/helenmasters/openjdk-tests/TestConfig/scripts/testKitGen/../../../autoGenTest.mk!

Warning: cannot find spec aix_ppc in ModesDictionaryService or ottawa.csv file!

Warning: cannot find spec aix_ppc-64 in ModesDictionaryService or ottawa.csv file!

Warning: cannot find spec aix_ppc-64_purec in ModesDictionaryService or ottawa.csv file!

Warning: cannot find spec aix_ppc_purec in ModesDictionaryService or ottawa.csv file!

Warning: cannot find spec linux_390 in ModesDictionaryService or ottawa.csv file!

Warning: cannot find spec linux_390-64 in ModesDictionaryService or ottawa.csv file!

Warning: cannot find spec linux_390-64_cs in ModesDictionaryService or ottawa.csv file!

Warning: cannot find spec linux_390-64_purec in ModesDictionaryService or ottawa.csv file!

Warning: cannot find spec linux_390_purec in ModesDictionaryService or ottawa.csv file!

Warning: cannot find spec linux_arm in ModesDictionaryService or ottawa.csv file!

Warning: cannot find spec linux_ppc-64_le in ModesDictionaryService or ottawa.csv file!

Warning: cannot find spec linux_ppc-64_le_purec in ModesDictionaryService or ottawa.csv file!

Warning: cannot find spec linux_x86 in ModesDictionaryService or ottawa.csv file!

Warning: cannot find spec linux_x86-64 in ModesDictionaryService or ottawa.csv file!

Warning: cannot find spec linux_x86-64_purec in ModesDictionaryService or ottawa.csv file!

Warning: cannot find spec linux_x86_purec in ModesDictionaryService or ottawa.csv file!

Warning: cannot find spec win_x86 in ModesDictionaryService or ottawa.csv file!

Warning: cannot find spec win_x86-64 in ModesDictionaryService or ottawa.csv file!

Warning: cannot find spec win_x86-64_purec in ModesDictionaryService or ottawa.csv file!

Warning: cannot find spec win_x86_purec in ModesDictionaryService or ottawa.csv file!

Warning: cannot find spec zos_390 in ModesDictionaryService or ottawa.csv file!

Warning: cannot find spec zos_390-64 in ModesDictionaryService or ottawa.csv file!

Warning: cannot find spec zos_390-64_purec in ModesDictionaryService or ottawa.csv file!

Warning: cannot find spec zos_390_purec in ModesDictionaryService or ottawa.csv file!

Generated /Users/helenmasters/openjdk-tests/TestConfig/scripts/testKitGen/../../../TestConfig/specToPlat.mk!
Generated /Users/helenmasters/openjdk-tests/TestConfig/scripts/testKitGen/../../../TestConfig/jvmTest.mk!

TEST AUTO GEN SUCCESSFUL
```

**Comment by helenmasters** [comment]
Also - the generated files don't have ! marks on the end of them so the messages referencing these files shouldn't either as it is confusing:

```
Generated /Users/helenmasters/openjdk-tests/TestConfig/scripts/testKitGen/../../../TestConfig/specToPlat.mk!
Generated /Users/helenmasters/openjdk-tests/TestConfig/scripts/testKitGen/../../../TestConfig/jvmTest.mk!
```

**Comment by neomatrix369** [comment]
Not sure if those warnings can be muted - they are not thrown by the `make` command

**Comment by smlambert** [comment]
Definitely Agree.  We should quiet the warnings when running in a typical scenario (probably can keep and print them when running in 'verbose' mode).

**Comment by renfeiw** [comment]
This issue is addressed in #14
